docs/administration/back_office/back_office_elements/reusable_components.md

```diff
@@ -132,51 +130,178 @@ The table component consists of the following blocks:
+
+You can set headline title using the `results_headline` macro, and generate different headlines based on parameters:
+
+- `count` - of all results, not only displayed on the first page
+- `has_filters` - when using filters
+- `phrase` - filtering phrase
+- `results_headline` - ensures the headlines consistency across the platform
+- `head_cols` - an array for table header (not headline), corresponds with consecutive column
+
+For table header there are the following column types:
+
+- normal content column `{ content: col_name }` (content is the title of the column)
+- icon column `{ has_icon: true }`
+- checkbox column `{ has_checkbox: true }`
+- action buttons column ` {  }` with additional parameters available for all of the objects mentioned earlier:
+ 
+    - class (CSS class)
+    - attr (HTML attributes)
+
+See the example:
+
+```html+twig
+{
+    content: 'foo',
+    class: 'bar',
+    attr: {
+        colspan: 2,
+    },
 
-    {% block tbody %}
-        {% for i in 1..10 %}
-            <tr>
-                <td></td>
-                <td>A{{ i }}</td>
-                <td>B{{ i }}</td>
-                <td>C{{ i }}</td>
-            </tr>
-        {% endfor %}
-    {% endblock %}
-{% endembed %}
 ```
 
+- `empty_table_info_text` and `empty_table_action_text` specify texts which are displayed when the table is empty.
+
+
+```html+twig
+{{ form_start(form_remove, {
+    action: path('ibexa.article.remove'),
+    attr: { class: 'ibexa-toggle-btn-state', 'data-toggle-button-id': '#article_remove_remove' }
+}) }}
+{% include '@ibexadesign/ui/component/table/table.html.twig' with {
+    headline: results_headline(pager.getNbResults()),
+    head_cols: [
+        { has_checkbox: true },
+        { has_icon: true },
+        { content: 'article.list.name'|trans|desc('Name') },
+        { content: 'article.list.content_type'|trans|desc('Content Type') },
+        { },
+    ],
+    body_rows,
+    actions: form_widget(form_remove.remove, { attr:
+        {
+            class: 'btn ibexa-btn ibexa-btn--ghost ibexa-btn--small',
+            disabled: true,
+        }}),
+    empty_table_info_text: 'article.list.empty'|trans|desc('You have no articles yet. Your articles will show up here.'),
+    empty_table_action_text: 'article.list.empty_desc'|trans|desc('Articles you create will show up here.'),
+} %}
+{{ form_end(form_remove) }}
+```
+
+Other table component parameters include:
+
+- `class` - (CSS table class)
+- `attr` - (other HTML attributes applied on the HTML table element), for example:
+    - `attr: { 'data-some-data-attribute-you-need': 'foo' }`
+- `table_body_class` and `table_body_attr` are the same as mentioned earlier, but applied on the table element
+- `show_head_cols_if_empty` - (default: `false`), by default, when `body_rows` is empty, the table component does not show the table header,it can be useful, for example, when rows are rendered dynamically with JavaScript on the browser side.
```

Comment:
```suggestion
- `show_head_cols_if_empty` - (default: `false`), by default, when `body_rows` is empty, the table component does not show the table header, but we may want to have it because for example rows are rendered dynamically with JavaScript on the browser side.
```